librecores-ci-riscv/Dockerfile

```diff
@@ -20,73 +20,17 @@
 #  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 #  THE SOFTWARE.
 
-FROM ubuntu:16.04
+FROM librecores/librecores-ci:latest
```

Comment:
pin the release version maybe? We will suffer from race conditions if we keep latest here